### ISSUE TITLE
Customer validator support

### ISSUE THREAD

**Issue by SirMullich** [circe/circe-json-schema#23]
Add support of custom validators. Currently, Everit supports [custom validators](https://github.com/everit-org/json-schema#example)

**Comment by SirMullich** [comment]
Suggestions are welcome for #22

**Comment by travisbrown** [comment]
I'm reluctant to move forward with this given #8 and the fact that this introduces _closer_ coupling with Everit. Circe already provides extensive facilities for validation, and the primary goal of this library is to support JSON Schema validation—not to be a one-stop validation solution.

**Comment by SirMullich** [comment]
I agree that this introduces closer coupling with Everit. Should I close this issue for now?
